.github/workflows/codeql.yml

```diff
@@ -0,0 +1,31 @@
+name: CodeQL
+
+on: 
+  schedule: 
+    - cron: '0 4 * * 1-5'
```

Comment:
It may also be good to be able to trigger a scan manually using the `workflow_dispatch` trigger

Reply:
second the above